charts/scalardb-cluster/README.md

```diff
@@ -63,12 +62,6 @@ Current chart version is `1.0.0-SNAPSHOT`
 | scalardbCluster.serviceMonitor.enabled | bool | `false` | Enable metrics collect with prometheus. |
 | scalardbCluster.serviceMonitor.interval | string | `"15s"` | Custom interval to retrieve the metrics. |
 | scalardbCluster.serviceMonitor.namespace | string | `"monitoring"` | Which namespace prometheus is located. by default monitoring. |
-| scalardbCluster.storageConfiguration.contactPoints | string | `"cassandra"` | The database contanct point such as a hostname of Cassandra or a URL of Cosmos DB account. |
-| scalardbCluster.storageConfiguration.contactPort | int | `9042` | The database port number. |
-| scalardbCluster.storageConfiguration.dbLogLevel | string | `"INFO"` | The log level of ScalarDB Cluster |
-| scalardbCluster.storageConfiguration.password | string | `"cassandra"` | The password of the database. For Cosmos DB, Dynamo DB please specify a secret key here. |
-| scalardbCluster.storageConfiguration.storage | string | `"cassandra"` | Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set. |
-| scalardbCluster.storageConfiguration.username | string | `"cassandra"` | The username of the database. For Cosmos DB please leave blank. For Dynamo DB please specify key id here. |
```

Comment:
The `scalardbCluster.storageConfiguration.*` configurations are deprecated. This is because, this parameter cannot set all properties of the ScalarDB Cluster.

Instead of that, we recommend using `scalardbCluster.scalardbClusterNodeProperties` that we can set all properties of the ScalarDB Cluster.